src/other/index.jl

```diff
@@ -128,8 +138,8 @@ function Base.push!(x::Index, nm::Symbol)
     return x
 end
 
-function Base.merge!(x::Index, y::AbstractIndex; makeunique::Bool=false)
-    adds = add_names(x, y, makeunique=makeunique)
+function Base.merge!(x::Index, y::AbstractIndex; makeunique::Bool=false, mergeduplicates=nothing)
```

Comment:
this function for sure does not need `mergeduplicates` kwarg as it does not take a data frame as a source